docs/developers/api.md

```diff
@@ -0,0 +1,223 @@
+- `public_title` - `string`
+- `target_sample_size` - `integer`
+- `gender` - `string`
+- `status` - `string`
+- `has_published_results` - `boolean`
+- `recruitment_status` - `string`
+- `registration_date` - `date`
+- `completion_date` - `date`
+
+
+- `interventions` - `object`:
+    - `id` - `string`
+    - `name` - `string`
+- `intervention` - `string`: maps to `interventions.name`
+
+
+- `conditions` - `object`:
+    - `id` - `string`
+    - `name` - `string`
+-  `condition` - `string`: maps to `conditions.name`
+
+
+- `locations` - `object`:
+    - `id` - `string`
+    - `name` - `string`
+    - `type`- `string`
+    - `role` - `string`
+- `location` - `string`: maps to `locations.name`
+
+
+- `persons` - `object`: 
+    - `id` - `string`
+    - `name` - `string`
+    - `role` - `string`
+- `person` - `string`: maps to `persons.name`
+
+
+- `organisations` - `object`: 
+    - `id` - `string`
+    - `name` - `string`
+    - `role` - `string`
+- `organisation` - `string`: maps to `organisations.name`
+
+
+- `publications` - `object`: 
+    - `id` - `string`
+    - `title` - `string`
+- `publication` - `string`: maps to `publications.title`
+
+
+- `documents` - `object`:
+    - `id` - `string`
+    - `name` - `string`
+    - `source_id` - `string`
+    - `source_url` - `string`
+    - `type` - `string`
+    
+    - `fda_approval` - `object`:
+        - `id` - `string`
+        - `action_date` - `date`
+        - `notes` - `string`
+        - `supplement_number` - [`long`](https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html)
+        - `type` - `string`
+        - `fda_application` - `object`:
+            - `id` - `string`
+            - `type` - `string`
+            - `drug_name` - `string`
+            - `active_ingredients` - `string`
+    -  `file` - `object`:
+        - `id` - `string`
+        - `documentcloud_id` - `string`
+        - `source_url` - `string`
+
+#### FDA documents search
+
+Here is a list of the attributes included in the `FDADocuments` index:
+
+- `id` - `string`
+- `name` - `string`
+- `source_id` - `string`
+- `source_url` - `string`
+- `type` - `string`
+
+- `fda_approval` - `object`:
+    - `id` - `string`
+    - `action_date` - `date`
+    - `notes` - `string`
+    - `supplement_number` - [`long`](https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html)
+    - `type` - `string`
+    - `fda_application` - `object`:
+        - `id` - `string`
+        - `type` - `string`
+        - `drug_name` - `string`
+        - `active_ingredients` - `string`
+
+-  `file` - `object`:
+    - `id` - `string`
+    - `documentcloud_id` - `string`
+    - `source_url` - `string`
+    - `pages` - `string` - raw text gathered with optical character recognition from files
+
+- `trial` - `object`:
+    - `id` - `string`
+    - `public_title` - `string`
+
+#### Examples
+
+* Simple attribute query 
+
+    To get all the `trials` directed at women:
+    ```
+    /v1/search?q=gender:female
```

Comment:
might be useful to include the HTTP verb used, such as:

    GET /v1/search?q=gender:female

(if you do this here, do it everywhere)